tests/functional/services/policy_engine/vulnerability_data_tests/test_vulnerability_scanner.py

```diff
@@ -93,6 +93,8 @@ def test_get_vulnerabilities_content(
 
         # compare only the vulnerability matches
         actual_results = actual_report["results"]
+        print(json.dumps(actual_results, indent=2))
```

Comment:
intentional addition?

Reply:
Print itself wasn't intentional. Will delete it on rebase